canvas_course_info/settings/aws.py

```diff
@@ -112,33 +111,32 @@
 # GUNICORN_WORKERS = SECURE_SETTINGS.get('gunicorn_workers', 4)
 # GUNICORN_TIMEOUT = SECURE_SETTINGS.get('gunicorn_timeout', 60)
 
-# Check if we want to do it like this or with the host and port 
-REDIS_URL = SECURE_SETTINGS.get('redis_url')
-
-LTI_REQUEST_VALIDATOR = 'course_info.validator.LTIRequestValidator'
+# Cache
+# https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-CACHES
 
-LTI_OAUTH_CREDENTIALS = SECURE_SETTINGS.get('lti_oath_credentials')
+# Check if we want to do it like this or with the host and port
+REDIS_URL = SECURE_SETTINGS.get('redis_url')
 
-# LTI_OAUTH_CREDENTIALS = {
-#      SECURE_SETTINGS.get('lti_oauth_course_info_consumer_key') :
-#          SECURE_SETTINGS.get('lti_oauth_course_info_consumer_secret')
-# }
+CACHES = {
+    'default': {
+        'BACKEND': 'redis_cache.RedisCache',
+        'LOCATION': "redis://%s/0" % (REDIS_URL),
```

Comment:
Well, it had the port in there too. `REDIS_URL` points to an existing (combined host-port) key in the secure config, so rather than update the config with new keys in all environments, just keeping it as-is.